src/shared-components/posts/createPostModal/createPostModal.module.css

```
@@ -257,11 +171,12 @@
 .loader {
   display: inline-block;
-  width: 16px;
-  height: 16px;
-  border: 2px solid rgba(255, 255, 255, 0.2);
-  border-top: 2px solid currentColor;
-  border-radius: 50%;
+  width: var(--space-5);
+  height: var(--space-5);
+  border: var(--border-2) solid
+    color-mix(in srgb, var(--color-white) 20%, transparent);
+  border-top: var(--border-2) solid currentColor;
```

Stylelint (17.3.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)